portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -1782,6 +2056,69 @@ definition {
 		}
 	}
 
+	@description = "LPS-122873 When the user flip the audit graph axes, the relative ordering of the categories remain the same"
+	@priority = "5"
+	test ReversingTheAxesUsingTheFlipButton {
```

Comment:
This test is not needed, there is already a test testing this, called FlipAxesButton. You can delete it.